src/components/json-schema-designer/item-details/item-details.component.tsx

```diff
@@ -321,16 +321,6 @@ export class ItemDetailsComponent {
               </div>
             </div>
           </div>
-          <div class="form-group">
```

Comment:
I'm not sure if we want to throw this away in all cases as this is a valid option for schemas. Do you want to add this as an option for the component?

Reply:
I talked with Payvand before removing it, and he agreed that if someone wanted to get that technical with objects, they can update the JSON on the JSON tab as they have to be that technical to set up additional properties that way in a blank text area anyways. If they do that, it keeps the "allow additional properties" checkbox checked too